server/routes/auth.go

```diff
@@ -121,7 +121,9 @@ func authenticateCb(ctx context.Context, oauthCfg *oauth2.Config, provider *oidc
 	return func(c echo.Context) error {
 		user, err := auth.ExchangeCode(ctx, c.Request(), oauthCfg, provider)
 		if err != nil {
-			return err
+			// Redirect to login with error param to show to users
```

Comment:
Is there a scenario where we get to this error because they need to sign up? And if so, is there a link on the log in page to sign up?

Reply:
There's a few scenario's that could get to this point. A user is added to the auth0 org but didn't click the email invite link. A user used the wrong gmail email when logging in. A user already clicked the invite link, didn't log in, and then clicked it again (invite already used). A user wasn't added to the auth0 org.

So there's a bunch of scenarios to handle, I think I generic 'Access Denied: Please confirm you have received and accepted the invitation to this namespace' message is probably best plus any additional error description from Auth0

Reply:
There's no sign up page (admin needs to add you first) unless you are signing up as the admin

Reply:
Got it, thank you for the context!